helm/prometheus-rules/templates/alerting-rules/flux.rules.yml

```
@@ -114,3 +114,15 @@ spec:
         severity: page
         team: honeybadger
         topic: releng
+    - alert: GSFluxSuspendedForTooLong
```

```suggestion
    - alert: FluxKustomizationSuspendedForTooLong
```

I think this name is more aligned with the other flux alerts. In the description it will be clear which kustomization is suspended. WDYT?

---

do we only want to detect Kustomization CRs? The metric considers both, Kustomization and GitRepository CRs, so if we do not want to narrow it down maybe we could simply use the `FluxSuspendedForTooLong`?

---

@rossf7 ⬆️

---

@ljakimczuk Good catch we need to check gitrepo CRs too. 👍